server/planning/planning/service.py

```diff
@@ -0,0 +1,78 @@
+from eve.utils import date_to_str
+
+from planning.types import PlanningResourceModel
+from planning.common import WORKFLOW_STATE
+from planning.core.service import BasePlanningAsyncService
+
+
+class PlanningAsyncService(BasePlanningAsyncService[PlanningResourceModel]):
+    resource_name = "planning"
+
+    async def get_expired_items(self, expiry_datetime, spiked_planning_only=False):
```

Comment:
Same as with Events.get_expired_items method, would be good to have the type defined for the signature